
syntax = "proto3";

package mesop.components.button;

message ButtonType {
  string color = 1;
  bool disable_ripple = 2;
  bool disabled = 3;
  string on_click_handler_id = 4;
  int32 variant_index = 5;
}
